<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClaimDetail</class>
 <widget class="QDialog" name="ClaimDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рекламация</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер рекламации</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата поступления</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>От кого получено</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип рекламации</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>144</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Эксплуатирующая организация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>176</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Объект установки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>206</y>
     <width>180</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Краткое описание</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>16</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Гарантия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>48</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вероятная причина</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>79</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата устранения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>108</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Меры устранения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>137</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Файл ответа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>177</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Заключение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNumber">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFromWho">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDateClaim">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTypeClaim">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOrganiz">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>144</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leObjectInst">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>176</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDescript">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>206</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbGuarantee">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>13</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leReason">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>43</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDateRepair">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>73</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDoRepair">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>103</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFileAnswer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>133</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTextResult">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>173</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>621</width>
     <height>201</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Тип</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Заводской №</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="tbAddDevice">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbDeleteDevice">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список устройств для ремонта</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>ClaimDetail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
